src/Header.css

```diff
@@ -1,52 +1,61 @@
 * {
-    /*all elements to inherit same font*/
-    font-family: "Montserrat", sans-serif;
-  }
+  /*all elements to inherit same font*/
+  font-family: "Montserrat", sans-serif;
+}
+
+body {
+  margin: 0;
+}
 
-nav{
-    height: 50px;
-    border: 1px solid #f8dabb;
-    background-color: #f8dabb;
+nav {
+  position: fixed;
+  top: 0;
+  width: 100%;
+  height: 50px;
+  border: 1px solid #f8dabb;
+  background-color: #f8dabb;
 }
 
 .navbar a {
-    font-weight: bold;
+  font-weight: bold;
 }
 
 .navbar a:hover {
-    opacity: 1;
+  opacity: 1;
 }
 
-.div-header{
-    width: 80%;
-    height: 100%;
-    margin: 0 auto;
-    display: flex;
-    flex-direction: row;
-    justify-content: space-between;
-    align-items: center;
+.div-header {
+  width: 80%;
+  height: 100%;
+  margin: 0 auto;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
 }
 
 .div-header a:hover,
-.div-header a:active 
```

Comment:
I'm not sure if this is important but is the change in colour supposed to be only for hover? There is a container/rectangle that appears once a link is selected/active (see attached image), but I don't know where it comes from.
![Screenshot 2020-11-20 140029](https://user-images.githubusercontent.com/35570430/99839504-cba9d500-2b38-11eb-91f0-8d39fff5120a.jpg)

Reply:
i think its for hover and active